src/services/configuration/__tests__/configuration.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { IWizdomCache, IWizdomLocalstorageCache } from "../../caching/cache.interfaces";
+import { IHttpClient, IHttpClientResponse } from "../../../shared/httpclient.wrappers/http.interfaces";
+import { IWizdomContext } from "../../context/context.interfaces";
+import { GetWizdomConfiguration } from "../configuration";
+
+describe("WizdomTranslationServiceFactory", () => {
```

Comment:
The tests is for "Wizdom Configuration" and not the TranslationServiceFactory. You just forgot to change it.